frontend/public/reducers/monitoring.ts

```diff
@@ -27,7 +27,7 @@ export const enum SilenceStates {
   Pending = 'pending',
 }
 
-export const alertState = (a: Alert): AlertStates => a?.state;
+export const alertState = (a: Alert): AlertStates => _.toLower(a?.state) as AlertStates;
```

Comment:
Shouldn't need to convert to lower case (same as https://github.com/openshift/console/pull/5785/files#r452628730)